openreview/venue/process/invitation_edit_process.py

```diff
@@ -155,6 +162,12 @@ def post_invitation(note):
         if 'replyNumber' in invitation.edit['content']:
             content['replyNumber'] = { 'value': note.number }
 
+        if 'invitationPrefix' in invitation.edit['content']:
+            content['invitationPrefix'] = { 'value': note.invitations[0].replace('/-/', '/') + str(note.number) }
+
+        if 'replytoReplytoSignatures' in invitation.edit['content']:
+            content['replytoReplytoSignatures'] = { 'value': client.get_note(note.replyto).signatures[0] }             
```

Comment:
Do you already have `client.get_note(note.replyto)` this note since you got all submissions and all replies? I'm just thinking maybe it's possible to avoid the API call or if you need to get a fresh note

Reply:
Thanks for the catch! I fixed it.